Refresher/Patching/EbootPatcher.cs

```diff
@@ -332,6 +333,21 @@ public List<Message> Verify(string url, bool patchDigest)
             messages.Add(new Message(MessageLevel.Warning,
                                      "Could not find the digest in the EBOOT. Resulting EBOOT may still work depending on game."));
 
+        if (this.GenerateRpcs3Patch)
+        {
+            if (string.IsNullOrWhiteSpace(this.GameVersion))
```

Comment:
Come to think of it, do we really need the user to specify the game version? We already parse out the PARAM.SFO file so it's not too far-fetched to just automatically gather it.

Can be resolved in a follow-up PR though.